bin/submit-local

```diff
@@ -1,8 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -x
 
-SPARK_HOME=$(python -c "import pyspark; print(pyspark.__path__[0])") \
+export PYSPARK_PYTHON=python3
+export SPARK_HOME=$(python3 -c "import pyspark; print(pyspark.__path__[0])")
```

Comment:
It amazes me that this is still an issue in the Spark ecosystem...

Reply:
This is more of python2/python3 issue inside of Debian rather than a Spark issue. The default python interpreter is _still_ python 2 because of system scripts (and the conservative nature of the distribution), but I use python3 for the package installs.